src/openrct2/ride/gentle/meta/Maze.h

```diff
@@ -20,5 +20,6 @@ constexpr const RideTypeDescriptor MazeRTD =
     SET_FIELD(TrackPaintFunction, get_track_paint_function_maze),
     SET_FIELD(Flags, RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_SUPPORTS | RIDE_TYPE_FLAG_HAS_SINGLE_PIECE_STATION | RIDE_TYPE_FLAG_NO_TEST_MODE | RIDE_TYPE_FLAG_NO_VEHICLES |
                      RIDE_TYPE_FLAG_TRACK_NO_WALLS | RIDE_TYPE_FLAG_IN_RIDE | RIDE_TYPE_FLAG_HAS_TRACK | RIDE_TYPE_FLAG_HAS_ENTRANCE_EXIT),
+    SET_FIELD(NameConvention, { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }),
```

Comment:
```suggestion
    SET_FIELD(NameConvention, { RIDE_COMPONENT_TYPE_TRAIN, RIDE_COMPONENT_TYPE_TRACK, RIDE_COMPONENT_TYPE_STATION }),
```
Indentation a bit weird compared to the others